src/importexport/mei/internal/meiconverter.cpp

```diff
@@ -0,0 +1,1846 @@
+    meiClef.SetDisPlace(meiStaffDef.GetClefDisPlace());
+    return Convert::clefFromMEI(meiClef, warning);
+}
+
+engraving::DurationType Convert::durFromMEI(const libmei::data_DURATION meiDuration, bool& warning)
+{
+    warning = false;
+    switch (meiDuration) {
+    case (libmei::DURATION_long): return engraving::DurationType::V_LONG;
+    case (libmei::DURATION_breve): return engraving::DurationType::V_BREVE;
+    case (libmei::DURATION_1): return engraving::DurationType::V_WHOLE;
+    case (libmei::DURATION_2): return engraving::DurationType::V_HALF;
+    case (libmei::DURATION_4): return engraving::DurationType::V_QUARTER;
+    case (libmei::DURATION_8): return engraving::DurationType::V_EIGHTH;
+    case (libmei::DURATION_16): return engraving::DurationType::V_16TH;
+    case (libmei::DURATION_32): return engraving::DurationType::V_32ND;
+    case (libmei::DURATION_64): return engraving::DurationType::V_64TH;
+    case (libmei::DURATION_128): return engraving::DurationType::V_128TH;
+    case (libmei::DURATION_256): return engraving::DurationType::V_256TH;
+    case (libmei::DURATION_512): return engraving::DurationType::V_512TH;
+    default:
+        warning = true;
+        return engraving::DurationType::V_QUARTER;
+    }
+}
+
+libmei::data_DURATION Convert::durToMEI(const engraving::DurationType duration)
+{
+    switch (duration) {
+    case (engraving::DurationType::V_LONG): return libmei::DURATION_long;
+    case (engraving::DurationType::V_BREVE): return libmei::DURATION_breve;
+    case (engraving::DurationType::V_WHOLE): return libmei::DURATION_1;
+    case (engraving::DurationType::V_HALF): return libmei::DURATION_2;
+    case (engraving::DurationType::V_QUARTER): return libmei::DURATION_4;
+    case (engraving::DurationType::V_EIGHTH): return libmei::DURATION_8;
+    case (engraving::DurationType::V_16TH): return libmei::DURATION_16;
+    case (engraving::DurationType::V_32ND): return libmei::DURATION_32;
+    case (engraving::DurationType::V_64TH): return libmei::DURATION_64;
+    case (engraving::DurationType::V_128TH): return libmei::DURATION_128;
+    case (engraving::DurationType::V_256TH): return libmei::DURATION_256;
+    case (engraving::DurationType::V_512TH): return libmei::DURATION_512;
+    default:
+        return libmei::DURATION_4;
+    }
+}
+
+void Convert::dynamFromMEI(engraving::Dynamic* dynamic, const StringList& meiLines, const libmei::Dynam& meiDynam, bool& warning)
+{
+    // The letters in the MEI to be mapped to SMuFL symbols in the xmlText
+    static std::map<Char, engraving::SymId> dynMap = {
+        { 'p', engraving::SymId::dynamicPiano },
+        { 'm', engraving::SymId::dynamicMezzo },
+        { 'f', engraving::SymId::dynamicForte },
+        { 'r', engraving::SymId::dynamicRinforzando },
+        { 's', engraving::SymId::dynamicSforzando },
+        { 'z', engraving::SymId::dynamicZ },
+        { 'n', engraving::SymId::dynamicNiente },
+    };
+
+    assert(dynamic);
```

Comment:
FYI: We have macro `IF_ASSERT_FAILED`  for such cases 
```
IF_ASSERT_FAILED(dynamic) {
      return;
}
```

The point is that an incorrect situation (with a null pointer) will also be processed in the release build, there will be no crash of the program.